### ISSUE TITLE
Is it possible to post the model trainned?

### ISSUE THREAD

**Issue by marchezinixd** [anjanatiha/Detection-of-Pneumonia-from-Chest-X-Ray-Images#1]
Hello,
I'm trying some CNN methods applied to x-ray images and found your repository. I did some nice tests here, but using networks with few layers to reduce the training time. I saw that you normaly save your models, can you send them here or by email, just to avoid the time of retrain the huge net. I'm using the same dataset.
Thank you.

**Comment by zahinahmed97** [comment]
yes Can I get the pretrained weights too? I'm doing a research using transfer learning and the weights would help a lot.
